test/src/test/java/hudson/model/RunMapTest.java

```diff
@@ -67,7 +68,7 @@
     @Test public void reloadWhileBuildIsInQueue() throws Exception {
         //logs.record(Queue.class, Level.FINE);
         FreeStyleProject p = r.createFreeStyleProject("p");
-        p.getBuildersList().add(new SleepBuilder(9999999));
+        p.getBuildersList().add(new SleepBuilder(Long.MAX_VALUE));
```

Comment:
Using `Long.MAX_VALUE` to be consistent with other tests and to more clearly indicate the intention that this is meant to run "forever" before being interrupted.